README.md

```diff
@@ -27,11 +29,26 @@ brew install docker-compose
 6. Enable privileged [testcontainers-dotnet](https://github.com/testcontainers/testcontainers-dotnet/issues/876#issuecomment-1930397928)  
 `echo "ryuk.container.privileged = true" >> $HOME/.testcontainers.properties`
 
+### Windows 
+
+#### Prerequisites
+
+- [Git](https://git-scm.com/download/win)
+- [.NET 8 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
+- [WSL2](https://docs.microsoft.com/en-us/windows/wsl/install) (To install, open a PowerShell admin window and run `wsl --install`)
+- [Virtual Machine Platform](https://support.microsoft.com/en-us/windows/enable-virtualization-on-windows-11-pcs-c5578302-6e43-4b4b-a449-8ced115f58e1) (Installs with WSL2, see link above)
+
 #### Installing Podman (Windows)
 
-TODO
+1. Install [Podman Desktop](https://podman.io/getting-started/installation).
+. 
+2. Start Podman Desktop and follow instructions to install Podman.
+
+3. Follow instructions in Podman Desktop to create and start a Podman machine.
+
+4. In Podman Desktop, go to Settings -> Resources and run setup for the Compose Extension. This will install docker-compose.
 
-#### Verify prerequisites
+### Running the project
 
 You can run the entire project locally using `podman compose`.
```

Comment:
Skulle vi begynne å bruke `docker-compose` istedenfor her nå?

Reply:
Nope, `podman compose up` roper automatisk på docker-compose, men sørger for at routingen og slikt oppfører seg pent med Podman.

Reply:
la til info om dette i dokumentasjonen.